e2e/helpers/pages/admin/whats-new/WhatsNewMenu.ts

```diff
@@ -0,0 +1,41 @@
+import {BasePage} from '../../BasePage';
+import {Page, Locator} from '@playwright/test';
+import {WhatsNewModal} from './WhatsNewModal';
+import {WhatsNewBadges} from './WhatsNewBadges';
+
+export class WhatsNewMenu extends BasePage {
+    readonly userMenuTrigger: Locator;
+    readonly whatsNewMenuItem: Locator;
+    private readonly badges: WhatsNewBadges;
+
+    constructor(page: Page) {
+        super(page, '/ghost');
+
+        this.userMenuTrigger = page.locator('[data-test-nav="arrow-down"]');
+        this.whatsNewMenuItem = page.locator('[data-test-nav="whatsnew"]');
+        this.badges = new WhatsNewBadges(page);
+    }
+
+    async openUserMenu(): Promise<void> {
+        await this.userMenuTrigger.click();
+        await this.whatsNewMenuItem.waitFor({state: 'visible'});
+    }
+
+    async openWhatsNewModal(): Promise<WhatsNewModal> {
+        await this.openUserMenu();
+        await this.whatsNewMenuItem.click();
+
+        const modal = new WhatsNewModal(this.page);
+        await modal.waitForModal();
+
+        return modal;
+    }
+
+    get avatarBadge(): Locator {
+        return this.badges.avatarBadge;
+    }
+
+    get menuBadge(): Locator {
+        return this.badges.menuBadge;
+    }
```

Comment:
I think we can add these as properties for the menu. The extra page object and getters add indirection.